hibernate-envers/src/main/java/org/hibernate/envers/strategy/AuditStrategy.java

```diff
@@ -199,4 +202,32 @@ void addAssociationAtRevisionRestriction(
 			String alias1,
 			boolean inclusive,
 			MiddleComponentData... componentDatas);
+
+	/**
+	 * Method to add additional Columns to the aud-tables.
+	 *
+	 * @param anyMapping the class mapping
+	 * @param revMapping the mapping of the revInfo-relation
+	 * @param auditEntitiesConfiguration the configuration of the audit entities
+	 */
+	default void addAdditionalColumns(
+			Element anyMapping,
+			Element revMapping,
+			AuditEntitiesConfiguration auditEntitiesConfiguration) {
+		//For backward compatibility
+	}
+
+	/**
+	 * Method do add additional initialization behavior
+	 *
+	 * @param revisionInfoClass the class of the revision info
+	 * @param revisionInfoTimestampData the timestamp of the revision info
+	 * @param serviceRegistry the service registry
+	 */
+	default void initialize(
+			Class<?> revisionInfoClass,
+			PropertyData revisionInfoTimestampData,
+			ServiceRegistry serviceRegistry) {
```

Comment:
I think the same holds true here for what I mentioned about `addAdditionalColumns`.